frontend/packages/topology/src/actions/modify-application.ts

```diff
@@ -1,9 +1,30 @@
+import { TFunction } from 'i18next';
+import { Action } from '@console/dynamic-plugin-sdk';
 import { KebabOption } from '@console/internal/components/utils';
 import { truncateMiddle } from '@console/internal/components/utils/truncate-middle';
 import { K8sResourceKind, K8sKind } from '@console/internal/module/k8s';
 import { RESOURCE_NAME_TRUNCATE_LENGTH } from '@console/shared/src/constants';
 import { editApplicationModal } from '@console/topology/src/components/modals';
 
+export const getModifyApplicationAction = (
+  kind: K8sKind,
+  obj: K8sResourceKind,
+  t: TFunction,
+): Action => {
+  return {
+    id: 'modify-application',
+    label: t('topology~Edit Application grouping'),
```

Comment:
You can actually use `i18next` directly and you wouldn't need to send t function.
```suggestion
    label: i18next.t('topology~Edit Application grouping'),
```